entry.html

```diff
@@ -2338,6 +2340,38 @@ <h5 class="mb-3"><label for="signature-data">Signaturdaten</label></h5>
 </script>
 
 
+<script>
+    const submitButtonToggler = {
+        // a global object to track whether the save button can be enabled
+        // and to toggle its state when needed
+        // we require data to be validated and only allow submitting after
+        // the local save has been done
+        dataValid: false,
+        dataSaved: false,
+        update: function () {
+            let submitButton = document.getElementById("submit-button");
+            if (this.dataValid && this.dataSaved) {
+                submitButton.removeAttribute("disabled");
+            } else {
+                submitButton.setAttribute("disabled", "");
+            }
+        },
+        toggle: function (property, value) {
```

Comment:
You might want to use `set` instead of `toggle`, since "toggle" ususally means to flip the logical state, i.e. to set `x := not x`, and you are just setting the value here.

Reply:
On the same note: `submitButtonToggler` should probably be something like `submitButtonUpdater`

Reply:
I'll go for a more explicit `setAndUpdate` to avoid collision with `set` keyword.